### PR TITLE
Dialog+tabs

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
Changes:

- DialogTabs
  - Changed name to TabbedDialog
  - Remove actions to be part of content; not sure about this
  - Make more params optional
  - Changed param names; part to follow convention, part opinion.. see what you think
  - Bunch of typo fixes
- Simplify fixture setup

https://deploy-preview-782--bldrs-share.netlify.app/cosmos/?fixtureId=%7B%22path%22%3A%22src%2FComponents%2FTabbedDialog.fixture.jsx%22%7D

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
image

**Comment by OlegMoshkovich** [comment]
image

image



Why remove the action button?

**Comment by OlegMoshkovich** [comment]
If we remove the action button from the dialog actions, we break MUI convention, and I thought we decided to stick to it, so it more cohesive.
image

**Review by OlegMoshkovich — COMMENTED** [review]
Much cleaner props!
Regarding the action button, if we remove it, we break MUI convention.
I thought we decided to stick to it, so it more cohesive. 
Having the button in the dialog actions also takes care of the spacing issue, that way it is always in the same place.

This is the design using MUI current system.
https://www.figma.com/file/DqKVaQEukf8vVHnlwiRGhv/Screens?type=design&node-id=2362-16501&mode=design&t=q4QyRjLNsdaT4r9i-0
What do you think about implementing it exactly as it is specified?

This is the master component sheet for the dialog:
https://www.figma.com/file/0WnF8npo2MGNmSgKKtnPuo/MUI-for-Figma-Material-UI-v5.14.0?type=design&node-id=6586-47137&mode=design&t=1s6f2SCSWGYpzfkk-0

**Comment by pablo-mayrgundter** [comment]
Ok, moved DialogActions back in.  PTAL

**Comment by OlegMoshkovich** [comment]
title disappeared.

**Review by OlegMoshkovich — CHANGES_REQUESTED** [review]
Several comments.

File: src/Components/TabbedDialog.fixture.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        (<p key='2'>{loremIpsum(2)}</p>),
+        (<p key='3'>{loremIpsum(4)}</p>),
+      ]}
+      actionCbs={[
+        () => debug().log('clicked 1'),
+        () => debug().log('clicked 2'),
+        () => debug().log('clicked 3'),
+      ]}
+      isDialogDisplayed={true}
+      setIsDialogDisplayed={() => debug().log('setIsDialogDisplayed')}
```
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply src/Components/TabbedDialog.fixture.jsx:28]: Done

**Comment by pablo-mayrgundter** [comment]
PTAL

**Review by OlegMoshkovich — APPROVED** [review]

**Comment by pablo-mayrgundter** [comment]
Cypress is either stuck or taking too long to run.  I just ran locally no issues.  Submitting.

**Pull request merged**